### DISCUSSION TITLE
GoldHEN Cheats Manager v0.7.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/PS4CheatsManager#26] [Announcements]
### Added

- Installed Game list filter to show only available cheats
- Offline cheat pack installation (.zip) from USB/HDD
  - `/mnt/usb0/GoldHEN_Cheat_Repository-main.zip`
  - `/data/GoldHEN_Cheat_Repository-main.zip`

<hr /><em>This discussion was created from the release <a href='https://github.com/GoldHEN/GoldHEN_Cheat_Manager/releases/tag/v0.7.0'>GoldHEN Cheats Manager v0.7.0</a>.</em>

**Comment by The-ELK1** [comment]
Please update Elden Ring cheats for v1.08 - obrigado/gracias

**Reply by The-ELK1** [reply]
Sorry, make it v.109 for CUSA28863!!

**Reply by bucanero** [reply]
You need to post your cheat requests here:
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/categories/cheat-requests

**Comment by The-ELK1** [comment]
Thanks. Done!
